modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -116,13 +116,21 @@ backend is ML2/OVS. There is no need for modifications if the backend is
 ML2/OVN.
 ====
 
-. To enforce network policies across Services, like when traffic goes through
+. In {rh-openstack} versions 13 and 15, to enforce
+network policies across Services, like when traffic goes through
 the Octavia load balancer, you must ensure Octavia creates the Amphora VM
-security groups on the user project. To do that, you must add the project ID
+security groups on the user project. Add the project ID
 to the `octavia.conf` configuration file after you create the project.
 +
-This ensures that required LoadBalancer security groups belong to that project
+This change ensures that required LoadBalancer security groups belong to that project,
 and that they can be updated to enforce Services isolation.
++
+[NOTE]
+====
+You do not need to perform this task in {rh-openstack} version 16 or later.
```

Comment:
s/You do not need to/Do not

> ! need to v
Avoid using “need to” to refer to a requirement that a product or procedure imposes. Use “